data/txHandler.go

```diff
@@ -224,6 +227,41 @@ func (handler *TxHandler) processIncomingTxn(rawmsg network.IncomingMessage) net
 	dec := protocol.NewMsgpDecoderBytes(rawmsg.Data)
 	ntx := 0
 	unverifiedTxGroup := make([]transactions.SignedTxn, 1)
+
```

Comment:
Since this does new work on every incoming `TX` we'll definitely want to run some cluster performance tests. The recent outbound-network-scheduler adding new work on every message wound up being untenable because it added just enough delay to every message to make the overall performance drop.

Reply:
Agreed!